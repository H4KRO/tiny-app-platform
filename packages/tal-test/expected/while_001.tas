var i = 0

while (i < 10) {
  log(i)
  set i = i + 1
}
