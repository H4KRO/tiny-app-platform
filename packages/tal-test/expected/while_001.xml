<program>
  <declarelocal>
    <literal />
  </declarelocal>
  <while>
    <binaryoperator>
      <local />
      <literal />
    </binaryoperator>
    <block>
      <call>
        <local />
        <local />
      </call>
      <assign>
        <local />
        <binaryoperator>
          <local />
          <literal />
        </binaryoperator>
      </assign>
    </block>
  </while>
</program>
